src/components/chat/w2wChat/searchBar/SearchBar.tsx

```diff
@@ -116,32 +125,55 @@ const SearchBar = () => {
         setHasUserBeenSearched(true);
       }
     } else {
-      setIsLoadingSearch(true);
-      const caip10 = w2wChatHelper.walletToCAIP10({ account: searchedUser, chainId });
-      let filteredData: User;
-      setHasUserBeenSearched(true);
-      if (searchedUser.length) {
-        filteredData = await PushNodeClient.getUser({ caip10 });
-        if (filteredData !== null) {
-          setFilteredUserData([filteredData]);
+      handleUserSearch(searchedUser);
+    }
+    setIsLoadingSearch(false);
+  };
+
+  async function handleUserSearch(userSearchData) {
```

Comment:
we can add function return type and param type

Reply:
will add parameter type and  add void return type as function not returning anything.